odl/contrib/pyshearlab/examples/wave_shear_separation.py

```diff
@@ -0,0 +1,51 @@
+import odl
```

Comment:
Needs a docstring that explains what happens in this example.

Reply:
Frankly this example is refusing to work for me and I don't really remember what it does, can I remove it?

Reply:
Ok, got it working, It'll stay for now.